recipes/pypardiso/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "pypardiso" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/haasad/PyPardisoProject/archive/refs/tags/v0.2.2.tar.gz
+  sha256: b1d3a139f8f04c74f8b2b5c4b55434e1f58b00ae3fe0ca975115971906d93eac
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  host:
+    - python
+    - setuptools
+    - numpy
+    - pip
+    - "libblas =*=*mkl"  # [win]
+    - mkl 2020.*  # [win]
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - "libblas =*=*mkl"
+    - scipy
+    - mkl  # [not win]
+    - mkl 2020.*  # [win]
```

Why does this need to be constrained on Windows?
```suggestion
    - mkl
```

---

The latest releases of MKL changed the name of the linked library, at least on Windows, so this is necessary to get the Windows version to build correctly.

---

I think you need `mkl-devel` and it should work

---

Yes; I updated the recipe.